gtk/CMakeLists.txt

```diff
@@ -158,22 +158,32 @@ target_compile_definitions(${TR_NAME}-gtk
     PRIVATE
         TRANSMISSIONLOCALEDIR="${CMAKE_INSTALL_FULL_LOCALEDIR}"
         GETTEXT_PACKAGE="${TR_NAME}-gtk"
-        G_DISABLE_DEPRECATED
-        GDK_PIXBUF_DISABLE_DEPRECATED
-        GDK_DISABLE_DEPRECATED
-        GTK_DISABLE_DEPRECATED
-        PANGO_DISABLE_DEPRECATED
         # FIXME: these break libnotify's headers
         # G_DISABLE_SINGLE_INCLUDES
         # GTK_DISABLE_SINGLE_INCLUDES
-        GDKMM_DISABLE_DEPRECATED
-        GIOMM_DISABLE_DEPRECATED
-        GLIBMM_DISABLE_DEPRECATED
-        GTKMM_DISABLE_DEPRECATED
-        PANGOMM_DISABLE_DEPRECATED
-        SIGCXX_DISABLE_DEPRECATED
         $<$<BOOL:${ENABLE_UTP}>:WITH_UTP>)
 
+if(DISABLE_DEPRECATED)
+	target_compile_definitions(${TR_NAME}-gtk
+	    PRIVATE
+		TRANSMISSIONLOCALEDIR="${CMAKE_INSTALL_FULL_LOCALEDIR}"
```

Comment:
Add one more indent lavel after `PRIVATE` as the rest of our CMake code.

Reply:
I don't understand the comment?

In the other places in `gtk/CMakeLists.txt` where `PRIVATE` is added, the private lines following it are indented four characters to the right of `PRIVATE` same as here, I think

Reply:
Just noticed there's a mix of tabs and spaces, and my tab width is set to 4 spaces, which is why I see it the way I do :)